index.html

```diff
@@ -4,12 +4,219 @@
+              <h5 class="money-text" >SERIES A</h5>
+            </div>
+            <div class="funding-links">
+              <a href="#" class="block">Telecommunications</a>
+              <a href="#" class="block">Expedition Growth Capital</a>
+            </div>
+            <div class="funding-desc">
+              <p>Ringover raises $12 million to bring enterprise communications to the cloud on VentureBeat
+              </p>
+            </div>
+          </div>
+
+          <div class="funding-element">
+            <img src="./images/image-funding.webp" alt="">
+            <div class="funding-h3">
+              <h3>Ringover</h3>
+            </div>
+            <div class="funding-money">
+              <h4 class="money-text">$12M</h4>
+              <h5 class="money-text" >SERIES A</h5>
+            </div>
+            <div class="funding-links">
+              <a href="#" class="block">Telecommunications</a>
+              <a href="#" class="block">Expedition Growth Capital</a>
+            </div>
+            <div class="funding-desc">
+              <p>Ringover raises $12 million to bring enterprise communications to the cloud on VentureBeat
+              </p>
+            </div>
+          </div>
+
+          <div class="funding-element fun-elem-4th">
+            <img src="./images/image-funding.webp" alt="">
+            <div class="funding-h3">
+              <h3>Ringover</h3>
+            </div>
+            <div class="funding-money">
+              <h4 class="money-text">$12M</h4>
+              <h5 class="money-text" >SERIES A</h5>
+            </div>
+            <div class="funding-links">
+              <a href="#" class="block">Telecommunications</a>
+              <a href="#" class="block">Expedition Growth Capital</a>
+            </div>
+            <div class="funding-desc">
+              <p>Ringover raises $12 million to bring enterprise communications to the cloud on VentureBeat
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </main>
+
+    <footer class="footer">
+      <div class="first-footer">
+        <ul class="social">
+          <li><a href="#"><i class="fab fa-facebook-square"></i></a></li>
+          <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+          <li><a href="#"><i class="fab fa-youtube"></i></a></li>
+          <li><a href="#"><i class="fab fa-instagram"></i></a></li>
+          <li><a href="#"><i class="fab fa-pinterest-p"></i></a></li>
+          <li><a href="#"><i class="fas fa-wifi"></i></a></li>
+          <li><a href="#"><i class="fas fa-envelope"></i></a></li>
+        </ul>
+
+        <ul class="second-ul">
+          <li><a href="#">EVENTS</a></li>
+          <li><a href="#">ABOUT</a></li>
+          <li><a href="#">ADVERTISE</a></li>
+          <li><a href="#">JOBS</a></li>
+          <li><a href="#">CONTACT</a></li>
+        </ul>
+      </div>
+
+      <div class="second-footer">
+        <div class="contant-footer flex">
```

Comment:
This one should be a `nav` html semantic tag

Reply:
done